terms/experimentalData/diagnosis.json

```diff
@@ -187,6 +187,36 @@
             "const": "traumatic brain injury",
             "description": "A form of acquired brain injury which occurs when a sudden trauma causes damage to the brain.",
             "source": "http://purl.obolibrary.org/obo/OAE_0002392"
+        },
+        {
+            "const": "lewy body dementia",
+            "description": "A progressive form of dementia characterized by the presence of protein deposits called Lewy bodies in the midbrain and cerebral cortex, and loss of cholinergic and dopaminergic neurons. The signs and symptoms overlap with Alzheimer's and Parkinson's disease.",
+            "source": "http://purl.obolibrary.org/obo/MONDO_0007488"
+        },
```

Comment:
We have "Lewy body dementia".

Reply:
Checked w/ Mette and removed the proposed "Lewy body dementia"; we can use "Lewy body disease".